spec/NewTwitchApi/NewTwitchApiSpec.php

```diff
@@ -4,46 +4,70 @@
 
 use GuzzleHttp\Client;
 use NewTwitchApi\Auth\OauthApi;
+use NewTwitchApi\Resources\AnalyticsApi;
+use NewTwitchApi\Resources\BitsApi;
+use NewTwitchApi\Resources\EntitlementsApi;
 use NewTwitchApi\Resources\GamesApi;
 use NewTwitchApi\Resources\StreamsApi;
+use NewTwitchApi\Resources\TagsApi;
 use NewTwitchApi\Resources\UsersApi;
 use NewTwitchApi\Resources\WebhooksApi;
 use NewTwitchApi\Webhooks\WebhooksSubscriptionApi;
 use PhpSpec\ObjectBehavior;
 
 class NewTwitchApiSpec extends ObjectBehavior
 {
-    function let(Client $guzzleClient)
+    public function let(Client $guzzleClient)
```

Comment:
Specs should not define a visibility for it's functions. Specs aren't considered "code". They're considered specs/documentation. Please remove the visibility identifiers from the spec methods.

More official explanation here: https://github.com/phpspec/phpspec/pull/590#issuecomment-67995169

Reply:
@echosa - I'm not sure what to do with this one. php-cs-fixer is actually making this change. I knew I didn't do this myself. Should I leave it or turn off php-cs-fixer to push the commit resolving this? In the future, if we run php-cs-fixer, this will fail.

Reply:
I've done this as requested in case the decision is to have them not defined on 1b4495f - I'll roll this back if you decide you want to revert this request

Reply:
@brandinarsenault php-cs-fixer doesn't do this for me. It should be limited to the non-spec namespaces in the `.php_cs.dist` file. How are you executing it? I recommend using the provided `Makefile`.

Reply:
I have it triggered on save through Atom's php-cs-fixer extension - I'll look into alternatives.